README.md

```diff
@@ -22,28 +22,60 @@ Our project consists of two main parts. The first part of our project is data vi
-* We will have an analysis of survey results that is useful in helping students find compatible teammates and interesting patterns or characteristics in the data that the instructor can use. The analysis will be backed up by research.
-* We will provide algorithms that can interpret the data in multiple ways.
+* Import Data: The foundation of our Teamwork Survey web app is the data, so we must have an easy way to put data in the database. When a user starts using the app, the database will be empty. The user’s only option will be to import data from a .csv file into the database, via a button on the main page. The web app assumes that the .csv file is set up such that it was an output from the [Teamwork Survey](https://docs.google.com/forms/d/1px6yEtC9zEG469OELr_1apL2iTUqwwDRYkoirncgVBI/edit) sent out by the muddersOnRails team. 
 
-## Architecture
+* Overall Distribution: Once students have been loaded into the database, a user can then examine the overall distribution of scores among them. Upon clicking “Overall distribution,” Python will generate a graph that Ruby will subsequently display for the user. The primary, secondary, tertiary, and quaternary types are displayed for each of the students in a split bar graph.
+
+* Individual Statistics: In approaching team-building, a user will want to look at which students they have and what those students’ types are. If a user has particular students in mind for a team or project, they can examine their types. On the "Individual Statistics" page, the user can select any number of students and see their scores. If more than six students are selected, an average will be displayed. Otherwise, the scores for each of the students will be displayed in a split bar graph. 
 
-The data processing will be done with Python using numpy. The processor will access the database which will be implemented in SQLite while we discuss data privacy concerns. Ruby will use the Python scripts to display the resulting graphs on a web page a user can access and add data too.
-There will be a simple database to hold JSON strings, some Python scripts that access the data to build graphs using numpy, and a Ruby page to display these graphs when asked for. Initially, there will only be one graph that Python needs to construct and Ruby needs to display.
+## Known Problems
 
-## Past Issues
+## Proposed Future Work
 
-None.
+* Allow user to filter students by course
+* Give the user the ability to save teams and compare them
+* Create a script that generates teams from a dataset based on scores and team work research
+* Method to save previous datasets when clearing out the database
 
-## Known Bugs
+## Contributing
 
-Currently does not work.
+1. Fork it!
+2. Create your feature branch: `git checkout -b my-new-feature`
+3. Commit your changes: `git commit -am 'Add some feature'`
+4. Push to the branch: `git push origin my-new-feature`
+5. Submit a pull request!
 
 ## References
```

Comment:
Are you getting rid of the whole reference section? If so, why is line 80 still there? If not, what kind of references are we putting in?